keith-interactive/src/browser/layered/layered-interactive-view.tsx

```diff
@@ -43,26 +44,27 @@ export function renderHierarchyLevel(nodes: KNode[], root: KNode): VNode {
         // determines whether only the layer constraint will be set when the node is released
         let onlyLC = shouldOnlyLCBeSet(selNode, layers, direction) && selNode.properties.layerId !== currentLayer
 
+        let curLayer = null
```

Comment:
@Drakae Why do you need curLayer. Why is currentLayer not enough? Since it is null per default?

Reply:
Maybe you need this since the currentLayer might be -1 or one higher than the existing layer and you want to return null instead in these cases.